dist/js/axs_testing.js

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
```

Comment:
Yeah agreed, that change is a byproduct of the fact that there was an initial change that involved nuking dist and adding it to gitignore, and we no longer want that, so I had to re-add it.

I'll just revert that file to its original version.